manifests/extension/apcu/params.pp

```diff
@@ -48,7 +48,7 @@
   ]
 
   if (versioncmp($php::params::major_version, "5.6") >= 0) {
-    $package  = "${php::params::non_versioned_extension_prefix}-apcu"
+    $package  = php::normalize_php_ext('apcu')
```

Comment:
Do we really need to care about php5.6 or lower or could we cleanup the code and remove the condition?

Reply:
We do